assets/src/dashboard/theme.js

```diff
@@ -72,6 +92,12 @@ const theme = {
       lineHeight: '18px',
       weight: '400',
     },
+    button: {
+      family: 'Google Sans, Sans Serif',
```

Comment:
Hmm did you mean `'Google Sans', sans-serif;` ?

Reply:
yup, typo thanks!